docs/reference/eql/requirements.asciidoc

```diff
@@ -34,3 +33,10 @@ A field containing the event classification, such as `process`, `file`, or
 Timestamp::
 A field containing the date and/or time the event occurred. This is typically
 mapped as a <<date,`date`>> field.
+
+[NOTE]
+====
```

Comment:
I don't know if this note is necessary in this section, as well.
Maybe have it reworded differently. Since the section is about required fields, how about something around "a <<nested,`nested`>> field or the sub-fields of a `nested` field cannot be used as a Timestamp or Event category".

Reply:
Thanks @astefan. I updated this note per your suggestion with 644d84d3302.